packages/code-du-travail-api/src/server/routes/idcc/idcc.elastic.js

```diff
@@ -1,42 +1,49 @@
 function getIdccBody({ query }) {
+  const matchPhrasePrefix = {
+    match_phrase_prefix: {
+      "title.french": {
+        query: `${query}`
+      }
+    }
+  };
+
+  const matchShortTitle = {
+    match: {
+      "shortTitle.french": {
+        query: `${query}`,
+        fuzziness: "1",
+        boost: 0.7
+      }
+    }
+  };
+
   return {
     size: 50,
-    _source: ["id", "title", "shortTitle", "url", "idcc", "slug"],
+    _source: ["id", "title", "shortTitle", "url", "idcc", "slug", "effectif"],
     query: {
       bool: {
         filter: [
           {
             term: {
               source: "conventions_collectives"
             }
-          }
+          },
+          matchPhrasePrefix,
```

Comment:
could you explain why you add it to the filter too  ?

Reply:
Now that we have the `effectif` based rank feature, any convention with effectif would match. 
So we need to ensure we only use effectif on convention matching the query for title or short title. 

But now that I look at it, it means docs _have to_ match the three filters, which is probably wrong.